tests/sentry/receivers/test_onboarding.py

```diff
@@ -981,3 +981,32 @@ def test_release_received_through_transaction_event(self):
             status=OnboardingTaskStatus.COMPLETE,
         )
         assert task is not None
+
+    def test_issue_alert_received_through_project_creation(self):
+        with self.feature("organizations:quick-start-updates"):
+            now = timezone.now()
+
+            first_organization = self.create_organization(owner=self.user, slug="first-org")
+            first_project = self.create_project(first_event=now, organization=first_organization)
+            # By default, the project creation will create a default rule
+            project_created.send(project=first_project, user=self.user, sender=type(first_project))
+            assert OrganizationOnboardingTask.objects.filter(
+                organization=first_project.organization,
+                task=OnboardingTask.ALERT_RULE,
+                status=OnboardingTaskStatus.COMPLETE,
+            ).exists()
+
+            second_organization = self.create_organization(owner=self.user, slug="second-org")
+            second_project = self.create_project(first_event=now, organization=second_organization)
+            # When creating a project, a user can opt out of creating a default rule
+            project_created.send(
+                project=second_project,
+                user=self.user,
+                sender=type(second_project),
+                default_rules=False,
+            )
```

Comment:
Is there a reason we are sending that message ourselves? The create_project factory should be able to fire that signal as well, no?

Reply:
> Is there a reason we are sending that message ourselves?

I noticed that in some cases we do not send the signal directly, but in this file, its manually. I’d prefer to keep it consistent.

> The create_project factory should be able to fire that signal as well, no?

Yes, I could pass a `fire_project_created=True` parameter, but I’d need to update the `project_created.send`   ([see here](https://github.com/getsentry/sentry/blob/45bd6e87cc45632e400934b2bf149b40e9d0407b/src/sentry/testutils/factories.py#L500-L519)) function to support dynamic `default_rules`. I can do that, but my intention was to keep this PR as simple as possible - without too many changes

Reply:
Sounds good